src/app/layout/qgslayoutdesignerdialog.cpp

```diff
@@ -4790,6 +4795,8 @@ void QgsLayoutDesignerDialog::toggleActions( bool layoutAvailable )
   mActionZoomOut->setEnabled( layoutAvailable );
   mActionZoomActual->setEnabled( layoutAvailable );
   mActionZoomToWidth->setEnabled( layoutAvailable );
+  mActionZoomLast->setEnabled( layoutAvailable );
```

Comment:
I think we'll need a QgsLayoutView::canZoomLast / canZoomNext, and then do:

```suggestion
  mActionZoomLast->setEnabled( layoutAvailable && mView->canZoomLast() );
```

Otherwise this will initially be enabled when there's nothing in the history for the actions

Reply:
Good point